canonical-web-and-design/ubuntu.com
# update cta copy on vmware takeover

## Done

- Updated CTA copy on VMWare to Openstack takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that CTA now reads "Read the whitepaper"
